ui/components/component-library/popover/popover.test.tsx

```diff
@@ -181,73 +181,110 @@ describe('Popover', () => {
       getByTestId('popover').querySelector('.mm-popover__arrow');
     expect(arrowElement).toHaveClass('mm-popover__arrow');
   });
-});
 
-test('should render Popover with isPortal set to false', () => {
-  const { getByTestId } = render(
-    <div>
+  test('should render Popover with isPortal set to false', () => {
```

Comment:
Non blocking: Any reason why we are using `test` here instead of `it`?  Seems like only tests in the extension that use `test` are the Popover and the Checkbox? Feel free to create another issue to address this later

<img width="324" alt="Screenshot 2023-11-09 at 7 37 20 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/aef6d13f-1957-40f0-b04d-cdf5266333ec"><img width="309" alt="Screenshot 2023-11-09 at 7 37 25 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/c1e2c9a3-fd37-4512-8188-ce68c0da9a87">